products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,210 @@
+---
+id: cli
+title: CLI
+pagination_label: CLI
+sidebar_label: CLI
+sidebar_position: 1
+sidebar_class_name: cli
+keywords: ['cli']
+description: Learn how to use the SailPoint CLI in this guide. 
+slug: /tools/cli
+tags: ['CLI']
+---
+
+## Start using the CLI
+
+Learn how to use the SailPoint command line interface (CLI) in this guide. You can use the CLI to interact with IDN and the rest of SailPoint's SaaS platform in a programmatic way. You can use the CLI to perform many functions you would have otherwise used Postman or custom scripts to perform before, and you can perform them directly on the command line with minimal setup. 
```

Comment:
We should remove the line about "and the rest of SailPoint's SaaS platforms" as that is not true today.